sbi/neural_nets/net_builders/flow.py

```diff
@@ -1085,7 +1128,26 @@ def build_zuko_flow(
         transforms = flow_built.transform.transforms
 
         z_score_x_bool, structured_x = z_score_parser(z_score_x)
-        if z_score_x_bool:
+
+        if z_score_x == "transform_to_unconstrained":
+            if x_dist is None:
+                raise ValueError(
+                    "Transformation to unconstrained space requires a distribution "
+                    "provided through `x_dist`."
+                )
+            elif not hasattr(x_dist, "support"):
+                raise ValueError(
+                    "`x_dist` requires a `.support` attribute for"
+                    "an unconstrained transformation."
+                )
+            else:
+                transform_to_unconstrained = mcmc_transform(x_dist)
+                transforms = (
+                    biject_transform_zuko(transform_to_unconstrained),
+                    *transforms,
+                )
```

Comment:
this is almost identical to the above if-case for CNF. Please double check whether it's possible to define a function "get_transform_to_unconstrained" that handles this.